packages/styles/scss/patterns/simplebenefits/_simplebenefits-item.scss

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin simplebenefits-item {
+  .#{$prefix}--simplebenefits__content-item {
+    &-container {
+      height: 100%;
+      padding-top: $carbon--layout-03;
+      @include carbon--breakpoint-down(lg) {
+        margin-left: 0;
+        margin-right: 0;
+        padding-left: $carbon--layout-01;
+        padding-right: $carbon--layout-01;
+      }
+      @include carbon--breakpoint(lg) {
+        padding-left: 0;
+        padding-right: 0;
+        margin-left: $carbon--layout-01;
+        margin-right: $carbon--layout-01;
+      }
+    }
+
+    &__title {
+      margin-bottom: $carbon--layout-03;
+      max-width: 80%;
+      @include carbon--type-style('expressive-heading-03');
+    }
+
+    &__devider {
```

Comment:
@emerson-pereira minor thing, `divider` is spelled wrong here...

Reply:
Fixed!